src/app/lib/business/logger/custom_console_output.dart

```diff
@@ -0,0 +1,14 @@
+import 'dart:developer';
+
+import 'package:logger/logger.dart';
+
+/// Workaround for colorization on iOS.
+/// See https://github.com/flutter/flutter/issues/64491 for more details.
+final class CustomConsoleOutput extends LogOutput {
```

Comment:
It feels like log outputs should be in the access layer because that's where we put I/O.

Reply:
It has been moved